### DISCUSSION TITLE
Error: HTTPSConnectionPool(host='infoasst-enrichmentweb-123abc.azurewebsites.net', port=443): Read timed out. (read timeout=60)

### DISCUSSION THREAD

**Discussion by jdnuckolls** [microsoft/PubSec-Info-Assistant#408] [General]
After deploying the last couple solutions, they both stopped working with the following error message.  Is this something that can be fixed without redeploying?
Error: HTTPSConnectionPool(host='infoasst-enrichmentweb-123abc.azurewebsites.net', port=443): Read timed out. (read timeout=60)

**Comment by ArpitaisAn0maly** [comment]
Hi [jdnuckolls](https://github.com/jdnuckolls), can you provide us more detail about error? like an error log. error message in your message is generic. we will need more context. Thank You.

**Reply by jdnuckolls** [reply]
I never could get it to resolve, so I ended up having to delete it all and redeploy.
I have it working now.  I just think I messed it up by deploying two solutions from the same GitHub repo.

**Comment by lon-tierney** [comment]
Closing discussion after speaking. Suspect environment bots shutting down services at night are related. If persistent and not related to service downgrade, please reopen @jdnuckolls
